src/test/java/CalendarServletTest.java

```diff
@@ -40,32 +49,42 @@
 
   private static final String EVENT_SUMMARY_ONE = "test event one";
   private static final String EVENT_SUMMARY_TWO = "test event two";
-  private static final String EVENT_ONE_TWO_JSON =
-      String.format(
-          "[{\"summary\":\"%s\"},{\"summary\":\"%s\"}]", EVENT_SUMMARY_ONE, EVENT_SUMMARY_TWO);
-  private static final String EVENT_ALL_JSON =
-      String.format(
-          "[{},{\"summary\":\"%s\"},{\"summary\":\"%s\"}]", EVENT_SUMMARY_ONE, EVENT_SUMMARY_TWO);
+  private static final int OFFSET_YEAR = 1900;
+  // private static final String EVENT_ONE_TWO_JSON =
+  //     String.format(
+  //         "[{\"summary\":\"%s\"},{\"summary\":\"%s\"}]", EVENT_SUMMARY_ONE, EVENT_SUMMARY_TWO);
+  // private static final String EVENT_ALL_JSON =
+  //     String.format(
+  //         "[{},{\"summary\":\"%s\"},{\"summary\":\"%s\"}]", EVENT_SUMMARY_ONE,
+  // EVENT_SUMMARY_TWO);
```

Comment:
Do you still need these constants? If not, please delete the comment.

Reply:
`Resolved`